platforms/Windows/sdk/sdk.wxs

```diff
@@ -105,7 +106,9 @@
                     </Directory>
                   </Directory>
                 </Directory>
-                <Directory Id="WindowsSDK_usr_share" Name="share" />
+                <Directory Id="WindowsSDK_usr_share" Name="share">
+                  <Directory Id="RuntimeMergeModules" Name="RuntimeMergeModules" />
```

Comment:
Could we root these as `%ProgramFiles64%\Swift\Redistributables\$(ProductVersion)\`?

Reply:
The per-user install root already has the version number:

    [LocalAppDataFolder]Programs\Swift\$(MajorMinorProductVersion)\

So would you prefer

    [LocalAppDataFolder]Programs\Swift\$(MajorMinorProductVersion)\Redistributables

or

    [LocalAppDataFolder]Programs\Swift\Redistributables\$(MajorMinorProductVersion)

or

    [LocalAppDataFolder]Programs\Swift\Redistributables\$(ProductVersion)

Reply:
I think that option 4 might be the safest option.  We do not currently have any gurantees about the ABI on any release on Windows (and I think may have broken compatibility on a patch release even).  As such, `[LocalAppDataFolder]Programs\Swift\Redistributables\$(ProductVersion)` seems most suitable at this time, but option 3 would be something that I imagine we would like to get to some time in the future.